examples/krm

```diff
@@ -12,12 +12,11 @@ The currently supported commands are:
 - kitty @ resize-os-window --height {height}: Set OS window's height
 """
 
+from argparse import ArgumentParser, Namespace
```

Comment:
The project organizes its imports by length, not alphabetically.

Reply:
I didn't organized imports manually. I used `isort` as I said before